.circleci/config.yml

```diff
@@ -58,7 +58,7 @@ jobs:
           command: |
             # Megaphone can take an arbitrary amount of time to start
             # up (probably waiting for mysql to accept connections)
-            x=0; until docker-compose run tests http --check-status "http://megaphone:8000/__heartbeat__" || [ $x -gt 10 ]; do date; sleep 1; echo $((x += 1)); done; if [ $x -eq 10 ]; then docker logs kintodist_megaphone_1; fi
+            x=0; until docker-compose run tests http --check-status "http://megaphone:8000/__heartbeat__" || [ $x -gt 10 ]; do date; sleep 1; echo $((x += 1)); done; if [ $x -gt 10 ]; then docker logs kintodist_megaphone_1; fi
```

Comment:
That's only 10 seconds which is a long time on a fast laptop but not necessarily in Circle.

Reply:
I had to test it and copied the line into a test bash script that did something else instead of docker-compose. Something that would always fail. And correctly, after about 10 seconds it did fail as expected.

Reply:
It's "at least" 10 seconds, since each `docker-compose run` command could take however long it takes to run.